app/services/pager_duty_sender.rb

```diff
@@ -0,0 +1,50 @@
+require 'net/http'
+require 'uri'
+
+class PagerDutySender
+  def self.handle_ticket_notification(ticket, notification_config)
+    if ticket.previously_new_record?
+      if !notification_config.filter? || notification_config.filter == :create
```

Comment:
I think we should have `notification_config.is_for_action?(:created)`

And then split the filters at "," and use include? In the array.

Reply:
Great idea!